src/backend/base/langflow/logging/logger.py

```diff
@@ -166,12 +171,36 @@ async def write_async(self, message):
         await asyncio.to_thread(self._sink.write, message)
 
 
+def is_valid_log_format(format_string) -> bool:
+    """Validates a logging format string by attempting to format it with a dummy LogRecord.
+
+    Args:
+        format_string (str): The format string to validate.
+
+    Returns:
+        bool: True if the format string is valid, False otherwise.
+    """
+    record = logging.LogRecord(
+        name="dummy", level=logging.INFO, pathname="dummy_path", lineno=0, msg="dummy message", args=None, exc_info=None
+    )
+
+    formatter = logging.Formatter(format_string)
+
+    try:
+        # Attempt to format the record
+        formatter.format(record)
+    except (KeyError, ValueError, TypeError):
+        return False
```

Comment:
I feel like we should log an error here so the user know why their log format isn't working.